cmd/krew/cmd/install.go

```diff
@@ -30,10 +32,45 @@ import (
 	"sigs.k8s.io/krew/pkg/installation"
 )
 
+// Downloads data from url to a filepath location
+func downloadFile(url string) (string, error) {
```

Comment:
Can you move that function into the `internal` package?

Reply:
Done